#+TITLE: GN1 Features

* Table of Contents 							:TOC:
 - [[#introduction][Introduction]]
 - [[#use-cases][Use cases]]
     - [[#explore-experiments-stored-in-the-database][Explore experiments stored in the database]]
     - [[#select-and-explore-each-phenotype-stored-in-an-experiment][Select and explore each phenotype stored in an experiment]]
     - [[#select-multiple-phenotypes][Select multiple phenotypes]]
     - [[#compare-multiple-experiments-for-genotype-by-environment][Compare multiple experiments for genotype by environment]]
 - [[#online-tour-and-tutorial][Online tour and tutorial]]
 - [[#documentation][Documentation]]
 - [[#search][Search]]
     - [[#species][Species]]
     - [[#group][Group]]
     - [[#data-type][Data type]]
     - [[#data-set][Data set]]
     - [[#get-any][Get any]]
     - [[#combined][Combined]]
     - [[#search-examples][Search examples]]

* Introduction

This document lists all the features attributed to GN1.

* Use cases

GN has many use cases. A non-comprehensive list includes:

** Explore experiments stored in the database
** Select and explore each phenotype stored in an experiment
*** Information on experiment, individuals and phenotype
*** Quick links to external web resources

When information is available one can select to visit external websites,
including UCSC, BioGPS, STRING, PANTHER, Gemma, SynDB, and Allen Brain Atlas (ABA).


*** Basic statistics for each phenotype

The basic statistics for each phenotype include probability plots and
bar graphs which allow checking the data for patterns and outliers.

*** Map QTL for the phenotype using different methods

Mapping methods includen in GN1 are

1. Interval mapping 
2. Marker regerssiong 
3. Composite mapping
4. Pair scan

each with with settings for including parents, weights and
permutations (when relevant). Before running the mapping GN 
checks for outliers and can remove them using [[https://en.wikipedia.org/wiki/Winsorising][winsorising]].

** Select multiple phenotypes
** Compare multiple experiments for genotype by environment

If two studies conducted used the same genotypes but different
individuals raised in different environments *agree* closely as judged
by a simple correlation coefficient between measurements across all
genotypes, then this tells you that the phenotype is robust for
numerous largely undefined environmental differences among
laboratories and cohorts. It also tells you that you are likely to be
dealing with a highly heritable trait that will be a good target for
genetic dissection and QTL mapping.

* Online tour and tutorial

GN1 comes with an online tour and tutorial. The tour gives an example
of selecting data based on a keyword search (a gene name) followed by
selecting expression probes that hits against the gene. Selecting
probes can give statistics, gene ontology type information and also
covariation of expression, correlations between probes and a host of
other functionality (see below).

* Documentation

GN1 comes with an online [[http://www.genenetwork.org/index4.html][user guide]].

A freely downloadable GN1 primer with a focus on exploring the genetic
control and analysis of behavioral variation using powerful online
resources is available from [[http://www.genenetwork.org/images/upload/Williams_Mulligan_Bioinformatics%20of%20Brain%20Short%202012.pdf][INTERNATIONAL REVIEW OF NEUROBIOLOGY]] (Vol
104, p135-158 2012 by Robert W. Williams and Megan K. Mulligan
Bioinformatics of Behavior: Part 2). Despite its specific audience it
is mostly generic. The generalized abstract could be read as:

This chapter provides an introduction into genetic control and
analysis of phenotype variation using powerful online resources. We
introduce you to the new field of systems genetics using case studies
that exploit populations of genetically diverse lines of mice. These
lines differ very widely in patterns of gene and protein expression
and in (classical) phenotypes. We address the following set of related
questions: (1) Can we combine massive genomic data sets with large
aggregates of precise quantitative phenotypical data? (2) Can we map
causal relations between gene variants and phenotypical differences?
(3) Can we simultaneously use these highly coherent data sets to
understand more about the underlying molecular and cellular basis of
these phenotypes.

* Search

When entering the first screen of the website a global search is
presented.

GN1 has a powerful search feature to mine for the experiments that are
included in the large database.

** Species

In the first field select the species. Options include human, model
organisms mouse, rat, Arabidopsis *thaliana* and others (such as
monkey, tomato and barley).

** Group

In the second field select the group information, usually a cross type for model
species or a tissue/experiment for the others.

** Data type

In the third field select the data type. Options include phenotype,
genotype (marker), and for Mouse (for example) a host of neurological
mRNA related data types (expression).

** Data set

In this field experimental sets can be selected for.

** Get any

This is the first search field which can look for simple terms using
wild cards.

** Combined

The combined search field allows for logical AND searches. The
searches allow looking for keywords (diabetes, nicotine), chromosome
locations (QTL, markers, genes, transcripts), GO terms, etc.

It is a bit tricky to find searches that match datasets. But once you
have a working selection it is also printed on the results page: E.g.,
GeneNetwork searched the GE-NIAAA Cerebellum mRNA M430v2 (May05) RMA
Database for all records with MEAN between 1 and 16 and with LRS
between 23 and 46. GeneNetwork found a total of 923 records.

# Note: a missed search leads to a somewhat ungratifying result

# Note: metadata in RDF would allow for creating sophisticated menus

Once you have a selection you can select phenotypes and (for example)
generate a QTL plot on the fly. 

** Search examples

On the main search screen there are some examples, but click the 
link to the [[http://www.genenetwork.org/index4.html][user guide]] to see full examples.

